modules/editor-service/src/main/java/org/opencastproject/editor/EditorServiceImpl.java

```diff
@@ -962,8 +997,12 @@ public EditingData getEditData(final String mediaPackageId) throws EditorService
 
       final int priority = thumbnailSourcePrimary.indexOf(track.getFlavor());
 
+      if (localPublication == null) {
+        localPublication = isLocal(track.getURI());
```

Comment:
This is calculated for each track, and would affect other tracks, right?

Cases:
- Initially null, afterwards it's not
- Initially *not* null, is not recalculated
- Next time someone calls `getEditData` the initial state of this would depend on what happened *last* time, right?

I mean, I guess it shouldn't change that often, but still...

Reply:
What is a case where this would change? Wither you have a shared file system in which case you have all the files on all servers, or you don't in which case all distribution files should be on the presentation server.